README.md

```diff
@@ -100,6 +100,18 @@ filtering. If you need a property that isn't supported. Feel free to PR.
                     history. If handled by the browser this path will be handled
                     like if you had provided it to the `href` attribute.
 
+* **className**  
+    *Type:*         `string`  
+    *Description:*  A css className to assign to the link.  This allows styling of the link
+    *Example*:
+    ```JSX
+    import Link from 'react-router-redux-dom-link'
+
+    export default const AboutLinkComponent () => (
+        <Link to="/about" replace className='navLink'>This link replaces the current URL</Link>
```

Comment:
I think a good Example only demos minimal code that is necessary to display the feature. No need for `replace` here.